platforms/documentation/docs/src/snippets/how-to/share-outputs/groovy/consumer/build.gradle

```diff
@@ -3,26 +3,31 @@ plugins {
     id("application")
 }
 
-configurations {
-    create("instrumentedRuntime") {
-        canBeConsumed = false
-        canBeResolved = true
-        attributes {
-            attribute(Usage.USAGE_ATTRIBUTE, objects.named(Usage, Usage.JAVA_RUNTIME))
-            attribute(LibraryElements.LIBRARY_ELEMENTS_ATTRIBUTE, objects.named(LibraryElements, "instrumented-jar"))
-        }
+// This configuration is used to declare dependencies only.
+def instrumentedRuntimeDependencies = configurations.dependencyScope("instrumentedRuntimeDependencies")
+
+// This resolvable configuration is used to resolve the instrumented JAR files.
+def instrumentedRuntime = configurations.resolvable("instrumentedRuntime") {
+    // Wire the dependency declarations
+    extendsFrom(instrumentedRuntimeDependencies.get())
+
+    // These attributes must be compatible with the producer
+    attributes {
+        attribute(LibraryElements.LIBRARY_ELEMENTS_ATTRIBUTE, objects.named(LibraryElements, "instrumented-jar"))
     }
 }
 // end::custom-config[]
 
 // tag::dependency[]
 dependencies {
-    add("instrumentedRuntime", project(":producer"))
+    // Declare a project dependency on the producer's instrumented output
+    add("instrumentedRuntimeDependencies", project(":producer"))
 }
 // end::dependency[]
 
 // tag::task[]
 tasks.register("runWithInstrumentation",JavaExec) {
+    // Use the resolved instrumented classpath
     classpath = configurations["instrumentedRuntime"]
```

Comment:
Should this be repalced with `instrumentedRuntime` (`def instrumentedRuntime` above)?